data-prepper-plugins/mutate-event-processors/src/main/java/org/opensearch/dataprepper/plugins/processor/mutateevent/CopyValueProcessorConfig.java

```diff
@@ -88,16 +86,16 @@ public Entry() {
     private List<Entry> entries;
 
     @JsonProperty(FROM_LIST_KEY)
-    @JsonPropertyDescription("The key of the list of objects to be copied. Either <code>from_key</code> and " +
-            "<code>to_key</code> or <code>from_list</code> and <code>to_list</code> must be defined.")
+    @JsonPropertyDescription("The key of the list of objects to be copied. " +
+            "Both <code>from_key</code> and <code>to_key</code> will be applied on the corresponding list.")
```

Comment:
Shall we specify at least one of them must be defined?

Reply:
I believe both have to be declared right? and I think maybe we should state this in from/to key. "Both from_key and to_key must be declared."